central/networkgraph/entity/datastore/datastore_bench_test.go

```diff
@@ -46,9 +46,7 @@ func BenchmarkNetEntityCreates(b *testing.B) {
 	entities, err := testutils.GenRandomExtSrcNetworkEntity(pkgNet.IPv4, b.N, testconsts.Cluster1)
 	require.NoError(b, err)
 
-	b.ResetTimer()
```

Comment:
Why removing `ResetTimer()`? Is it now contained within `b.Loop()`?
If not, we must persist it.